roma/romadom.php

```diff
@@ -2111,14 +2114,11 @@ protected function callGarage ( &$garageResult, $target) {
 	fwrite($handle, $this->SaveXML());
 	fclose($handle);
 	$this->updateProgressBar( '20' );
-	$cfile = new CURLFile($tmpfname,'application/xml');
-	$file = array('upload' => $cfile);
+	$file = array("upload"=>"@" . $tmpfname);
```

Comment:
This as well 
https://github.com/TEIC/Roma/commit/d4f1f52e9142204dba0954993ca2fea74c77fdfc